langchain4j-parent/pom.xml

```diff
@@ -19,6 +19,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.build.outputTimestamp>1714382357</project.build.outputTimestamp>
         <openai4j.version>0.22.0</openai4j.version>
+        <azure-ai-inference-api.version>1.0.0-beta.2</azure-ai-inference-api.version>
```

Comment:
What about keeping version numbers in the module pom instead? Otherwise every time version is changed, CI will have to do a complete build

Reply:
I was just copying on the other module, which one do you prefer?

Reply:
I would keep versions specific to the Azure in Azure module(s). Other versions will be moved to their respective modules as well soon.

Reply:
Done, thanks!